GG/src/Wnd.cpp

```diff
@@ -606,17 +606,25 @@ void Wnd::DetachChildren()
 
 void Wnd::DeleteChild(Wnd* wnd)
 {
-    if (wnd && std::find(m_children.begin(), m_children.end(), wnd) != m_children.end()) {
-        delete wnd;
+    if (wnd) {
```

Comment:
prefer

    if (!wnd)
        return

    // more code not indended unnecessarily